src/app.js

```diff
@@ -1,21 +1,32 @@
 'use strict';
 
-/**
- * Implement sum function:
- *
- * Function takes 2 numbers and returns their sum
- *
- * sum(1, 2) === 3
- * sum(1, 11) === 12
- *
- * @param {number} a
- * @param {number} b
- *
- * @return {number}
- */
-function sum(a, b) {
-  // write code here
-  return a + b;
-}
-
-module.exports = sum;
+const http = require('http');
+const fs = require('fs');
+
+const PORT = process.env.PORT || 3000;
+
+const server = http.createServer((req, res) => {
+  const { pathname } = new URL(req.url, `http://${req.headers.host}`);
+  const pathParts = pathname.slice(1).split('/');
+
+  if (pathParts[0] !== 'file') {
+    res.end('The file path must start with "/file/"');
+
+    return;
+  }
+
+  const pathToFile = pathParts.slice(1).join('/') || 'index.html';
+
+  fs.readFile(`./public/${pathToFile}`, (error, data) => {
```

Comment:
Using a relative path (./public/${pathToFile}) can expose files outside of the intended directory if the request contains '../' in the pathname. To resolve this issue, use the 'path' module to resolve the absolute path and make sure it still resides within the 'public' directory.